book/forms.rst

```diff
@@ -1048,7 +1051,8 @@ that will house the logic for building the task form::
             $builder
                 ->add('task')
                 ->add('dueDate', null, array('widget' => 'single_text'))
-                ->add('save', 'submit');
+                ->add('save', 'submit')
```

Comment:
I'm just not sure it would be a good idea just remove all of them out of the blue. Because even thought it's a best practice this topic on forms is kind of a special one. So I'm not sure it would be a good idea to remove the submit button in the beginning of the topic. But later especially when you start to move forms to classes I think they could be safely removed. Still at some point it might be a good idea to mention that best practice is not to add buttons.

I would guess the best choice would be for docs team to give a quick glance to a topic and decide on the most user friendly solution.